backend/canisters/user/api/src/lib.rs

```diff
@@ -142,3 +144,14 @@ pub struct DiamondMembershipPaymentReceived {
     pub recurring: bool,
     pub send_bot_message: bool,
 }
+
+pub fn map_chats_to_chat_ids(chats: Vec<Chat>) -> Vec<ChatId> {
```

Comment:
Probably better to add a chat_id function to Chat which returns Option<ChatId>

Reply:
Actually from the looks of it this is only a temporary thing needed for the switch from updates_v2